Source/Views/PageView.swift

```diff
@@ -79,24 +79,7 @@ class PageView: UIScrollView {
   // MARK: - Layout
 
   func configureImageView() {
-    guard let image = imageView.image else { return }
-
-    let imageViewSize = imageView.frame.size
-    let imageSize = image.size
-    let realImageViewSize: CGSize
-
-    if imageSize.width / imageSize.height > imageViewSize.width / imageViewSize.height {
```

Comment:
Actually there is a bug in the root view controller itself, not the Lightbox, like http://stackoverflow.com/questions/26037472/uiwindow-with-wrong-size-when-using-landscape-orientation